lib/files.com/api_client.rb

```diff
@@ -287,13 +287,13 @@ def execute_request_with_rescues(base_url, context, skip_body_logging = false)
         code: error_data[:code] || resp.http_status,
       }
 
-      return APIError.new(error_data[:message], opts) unless resp&.data&.dig(:type)
+      return APIError.new(error_data[:message], **opts) unless resp&.data&.dig(:type)
 
       begin
         error_class = Files.const_get(resp.data[:type].split("/").map { |piece| piece.split("-").map(&:capitalize).join('') + 'Error' }.join("::"))
         error_class.new(error_data[:message], opts)
```

Comment:
@zeisler - Hi! 👋 

I think `opts` on this line (294) needs the same treatment. Do you agree? If so, would you like another PR for that?

Reply:
Disregard. It looks like that got updated outside of this PR. Thanks 👋